### DISCUSSION TITLE
spdy http/2 不支持 node 15 +，而 umi4 的 https 又把开启 https 和 http/2 绑定到一起，无法只开启 https 不开启 http/2，导致 node 15+ 想开启 https 都不可以，这个有计划解决下吗？

### DISCUSSION THREAD

**Discussion by lvren** [umijs/umi#10247] [Q&A] [answered]
https 实现入口
![image](https://user-images.githubusercontent.com/11779753/211305833-9268819b-ea5d-4580-b02f-a0d527701d1f.png)

https://github.com/umijs/umi/blob/master/packages/bundler-utils/src/https.ts
![image](https://user-images.githubusercontent.com/11779753/211305729-74fa9b21-2a8f-4a32-9675-3bfea896a15c.png)

**Comment by fz6m** [comment, marked as answer]
只开启 https 找个代理工具就行，比如 charles 。

**Reply by lvren** [reply]
emmm，这不是还得再开个工具吗，明明是一个工具可以搞定的事情
